### DISCUSSION TITLE
Drop all `inverted` variations in favor of a proper dark theme?

### DISCUSSION THREAD

**Discussion by lubber-de** [fomantic/Fomantic-UI#2868] [Polls]
We added a lot of `inverted` variants for several components which were not already available in SUI. However there are still many missing and it also isn't completely consistent everywhere.

Removing all existing inverted variants would heavily reduce the codebase (yes, you already can disable the inverted variants inside https://github.com/fomantic/Fomantic-UI/blob/develop/src/themes/default/globals/variation.variables to get the same effect)

Thoughts:
- Instead of those inverted variants, we could make the usage of color options and a proper dark theme replace the inverted variants.
- A current possible mix of those would need additional color options instead, which might result in the same effort than keeping the codebase for inverted.
- Keeping the `inverted` logic would also mean to also support that in a dark theme (which might appear sooner or later anyway :wink:), an `inverted` variant inside a dark theme means to show the component in a light manner, so just as they look while using the default theme.
- Some components like `inverted button` provide a totally different style than just being "inverted", such component variants might need a renaming instead

I wonder how many people actually mix non-inverted and inverted variants?

**Comment by jamessampford** [comment]
By removing `inverted` variants, presumably this would just be those that are already being used as dark-theme like components?

The coloured inverted variants should remain, for example `black inverted segment`, though I’d imagine would need some thought around this as to how it would appear in a dark theme, still black or a different shade of black to be more distinct from the dark theme background?
